packages/devextreme/testing/tests/DevExpress.viz.sparklines/sparkline.tests.js

```diff
@@ -1433,6 +1433,17 @@ QUnit.begin(function() {
         ]);
     });
 
+    QUnit.test('winloss sparkline lossColor should change at runtime', function(assert) {
+        const sparkline = this.createSparkline({
+            type: 'winloss',
+            lossColor: 'blue',
+        });
+
+        sparkline.option({ lossColor: 'green' });
+
+        assert.strictEqual(sparkline.option('lossColor'), 'green', 'lossColor should be green after runtime change');
```

Comment:
this condition won't fail if you remove your fix, i suppose. probably we have to use 'checkCustomizePoint' here, please check